docs/proposals/observability.md

```diff
@@ -0,0 +1,311 @@
+
+The policy will only take effect if a [GatewaySettings](gateway-settings.md) configuration has been linked to the GatewayClass.
+
+### Creating the Effective Policy in NGINX Config
+
+To determine how to reliably and consistently create the effective policy in NGINX config, we need to apply the policies for each attachment scenario to the three NGINX mappings described [here](/docs/developer/mapping.md).
+
+The following examples use the `ClientSettingsPolicy`, but the rules are the same for the `ObservabilityPolicy`.
+
+A. Distinct Hostname:
+![example-a2](/docs/images/client-settings/example-a2.png)
+
+B. Same Hostname:
+![example-b2](/docs/images/client-settings/example-b2.png)
+
+C. Internal Redirect
+![example-c2](/docs/images/client-settings/example-c2.png)
+
+For this attachment scenario, specifying the directives in the _final_ location blocks generated from the HTTPRoute with the policy attached achieves the effective policy. _Final_ means the location that ultimately handles the request.
+
+## Use Cases
+
+- As an Application Developer, I want to enable observability -- such as tracing -- for traffic flowing to my application, so I can easily debug issues or understand the use of my application.
+
+## Testing
+
+- Unit tests
+- Functional tests that verify the attachment of the CRD to a Route or Route rule, and that NGINX behaves properly based on the configuration. This includes verifying tracing works as expected.
+
+## Security Considerations
+
+Validating all fields in the `ObservabilityPolicy` is critical to ensuring that the NGINX config generated by NGINX Gateway Fabric is correct and secure.
+
+All fields in the `ObservabilityPolicy` will be validated with Open API Schema. If the Open API Schema validation rules are not sufficient, we will use [CEL](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules).
+
+RBAC via the Kubernetes API server will ensure that only authorized users can update the CRD containing Gateway Settings.
+
+## Alternatives
+
+- Combine with OTel settings in Gateway Settings for one OTel Policy: Rather than splitting tracing across two Policies, we could create a single tracing Policy. The issue with this approach is that some tracing settings -- such as exporter endpoint -- should be restricted to Cluster Operators, while settings like attributes should be available to Application Developers. If we combine these settings, RBAC will not be sufficient to restrict access across the settings. We will have to disallow certain fields based on the resource the Policy is attached to. This is a bad user experience.
+- Inherited Policy: An Inherited Policy would be useful if there is a use case for the Cluster Operator enforcing or defaulting the OTel tracing settings included in this policy.
```

Comment:
if we get enough requests for this, will there be an upgrade path to support an inherited policy?

Reply:
Interesting question...are we technically breaking backwards compatibility in that case? Because when upgrading, the config from the old version will still work with the new version. The only difference is if the cluster operator adds a new policy, it will change behavior. It's a new feature that has to be opted-in to, nothing should break just by upgrading to the new version of NGF.